package.json

```diff
@@ -20,6 +20,7 @@
     "@mui/material": "^5.10.0",
     "axios": "^1.3.4",
     "bech32": "^2.0.0",
+    "json5": "^1.0.2",
```

Comment:
what is the reason to add this library?

Reply:
We need to bump the used revision to 1.0.2 to solve the following issue:
 - https://github.com/geniusyield/atlas-docs/security/dependabot/2

At the moment we are using a revision with a known vulnerability. 

Please make sure to update the dependency to a revision that contains the necessary patch. Thank you.